WFInfo/Data.cs

```diff
@@ -1319,10 +1317,8 @@ private void SendMessage(string data)
                 if (marketSocket.ReadyState == WebSocketState.Closed || marketSocket.ReadyState != WebSocketState.Open)
                 {
                     marketSocket.Close();
-                    OpenWebSocket();
+                    bool result = OpenWebSocket().Result;
```

Comment:
Is there specific reason we don't check this "result" output?

Reply:
The result is not used to verify the status of the task, rather to make sure the asynchronous task has finished before moving on. The previous way of handling it caused some errors to throw in edge cases, this way of handling it solves that.